docs/guide/tutorial-6-4-wheels-robot.md

```diff
@@ -131,29 +130,46 @@ In our case it's along the x-axis (so `(1, 0, 0)`).
+* **endPoint** : Add a [Solid](../reference/solid.md) node, then a [Shape](../reference/shape.md) node in the `children` field of the [Solid](../reference/solid.md), and finally, add a [Cylinder](../reference/cylinder.md) in the `geometry` field of the [Shape](../reference/shape.md) node. The [Cylinder](../reference/cylinder.md) should have a `radius` of `0.04` and a `height` of `0.02`.
 Set the color of the wheels to green.
-In order to be able to actuate the wheels, add a [RotationalMotor](../reference/rotationalmotor.md) to each [HingeJoint](../reference/hingejoint.md), and change their `name` fields from `wheel1` to `wheel4`.
-These labels will be used to reference the wheels from the controller.
+
 
 ### Sensors
 
+The sensors used on the e-puck of [Tutorial 4](tutorial-4-more-about-controllers.md) are not the same as those used in this section. Pay attention to their measuring interval which is between `0 cm = 0` and `10 cm = 1000`.
+Find more information about the `lookupTable` field [here](../reference/distancesensor.md#lookup-table).
+
+
 The last part of the robot modeling is to add the two distance sensors to the robot.
 This can be done by adding two [DistanceSensor](../reference/distancesensor.md) nodes as direct children of the [Robot](../reference/robot.md) node.
 Note that the distance sensor acquires its data along the +*x*-axis.
 So rotating the distance sensors in order to point their *x*-axis outside the robot is necessary (see the [figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices)).
 
+%spoiler "**Reminder**: How to know the orientation of the distance sensor?"
+
+As already says in [Tutorial 3](tutorial-3-appearance.md), the distance sensor rays can be viewed using the shortcut `(Ctrl + F10)` or `View / Optional Rendering / Show DistanceSensor Rays`.
+
+%end
+
 > **Hands on #5**: Add the two distance sensors as explained above.
 The distance sensors are at an angle to 0.3 [rad] with the robot front vector.
 Set their graphical and physical shape to a cube (not transformed) having a edge of `0.01` [m].
 Set their color to blue.
 Set their `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices).
 
+%spoiler "**Tips**: How to precisely orient distance sensors?"
+
+In the [DistanceSensor](../reference/distancesensor.md) node, the rotation field has 4 parameters. If you set the angle parameter at 0, then you can use the wheel's mouse to increment/decrement by step of 0.1309 rad (= 7.5 degree).
```

Comment:
If you set the angle parameter **to** 0.
wheel's mouse &rarr; mouse wheel
step &rarr; steps